### PR TITLE
Feature/285 - AZ Accordion

### PR DESCRIPTION
**Pull request by jwmoore1**
## Description
Adds basic accordion functionality to the Page content type in Quickstart.

## Related Issue
#285 

## How Has This Been Tested?
Although still a work in progress, this has been built and tested locally with Lando. Since this was based on the az_cards module, there is still code cleanup that needs to be done along with minor functionality improvements.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] I've added this Pull Request to the AZ Quickstart project in the right column.
- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by jwmoore1** [comment]
There are still a few things that could be done for the accordion:

- [ ] Make the Collapsed by Default field checked by default as in QS1
- [ ] Make the AZQS text formats available in the body field
- [ ] Code cleanup

**Comment by danahertzberg** [comment]
Looks like the Accordions aren't showing up on the front end anymore and the `Add another item` button isn't working

File: modules/custom/az_accordion/src/Plugin/Field/FieldWidget/AZAccordionWidget.php [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+      '#type' => 'textfield',
+      '#title' => $this->t('Accordion Title'),
+      '#default_value' => isset($items[$delta]->title) ? $items[$delta]->title : NULL,
+    ];
+
+    $element['body'] = [
+      '#type' => 'text_format',
+      '#title' => $this->t('Accordion Item'),
+      '#default_value' => isset($items[$delta]->body) ? $items[$delta]->body : NULL,
+      '#format' => $items[$delta]->body_format ?? 'basic_html',
```
Comment (danahertzberg) [inline modules/custom/az_accordion/src/Plugin/Field/FieldWidget/AZAccordionWidget.php:38]:
```suggestion
      '#format' => $items[$delta]->body_format ?? 'az_standard',
```

**Comment by danahertzberg** [comment]
> Looks like the Accordions aren't showing up on the front end anymore and the `Add another item` button isn't working

Now the `Add another item` button works, but the accordions still aren't appearing.

**Comment by tadean** [comment]
- Fixed a typo that caused the display of the accordion not to happen
- Changed the form items to be titled `Accordion Item Title` and `Accordion Item Body`

**Review by danahertzberg — APPROVED** [review]

**Review by camikazegreen — APPROVED** [review]

**Review by kevcooper — APPROVED** [review]

**Pull request merged**